Bottom: 72b24cdde0a69dba39cba62fab5c8386dfdd9772
Top:    091aa50ff741cdabd1887648a082adedf3968761
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-06-05 12:07:58 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index d6d7a40..34257c9 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ void ghid_set_lock_effects (hidGC gc, AnyObjectType *object);
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
